tools/releases/distros.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+GOARCH=( amd64 )
+
+# first component is the distribution name, second is the system - must map to
+# valid $GOOS values
+DISTRIBUTIONS=(debian:linux ubuntu:linux rhel:linux centos:linux darwin:darwin)
+
+
+BINTRAY_ENDPOINT="https://api.bintray.com/"
+BINTRAY_SUBJECT="kong"
+BINTRAY_REPOSITORY="kuma"
+
+
+function msg_green {
+  builtin echo -en "\033[1;32m"
+  echo "$@"
+  builtin echo -en "\033[0m"
+}
+
+
+function msg_red() {
+  builtin echo -en "\033[1;31m" >&2
+  echo "$@" >&2
+  builtin echo -en "\033[0m" >&2
+}
+
+
+function msg_err() {
+  msg_red $@
+  exit 1
+}
+
+
+function get_envoy() {
+  local distro=$1
+
+  local status=$(curl -L -o build/envoy-$distro -u $BINTRAY_USERNAME:$BINTRAY_API_KEY \
+    --write-out %{http_code} --silent --output /dev/null \
+    "https://kong.bintray.com/envoy/envoy-1.11.0-$distro")
+
+  [ "$status" -ne "200" ] && msg_err "Error: failed downloading Envoy"
+}
+
+
+function create_tarball {
+  local system=$1
+  local arch=$2
+  local distro=$3
+
+  local dest_dir=build/kuma-$system-$arch
+
+  rm -rf $dest_dir
+  mkdir $dest_dir
+  mkdir $dest_dir/bin
+  mkdir $dest_dir/conf
+
+  get_envoy $distro
+  chmod 755 build/envoy-$distro
+
+  cp -p build/envoy-$distro $dest_dir/bin/envoy
+  cp -p build/artifacts-$system-$arch/kuma-cp/kuma-cp $dest_dir/bin
+  cp -p build/artifacts-$system-$arch/kuma-dp/kuma-dp $dest_dir/bin
+  cp -p build/artifacts-$system-$arch/kumactl/kumactl $dest_dir/bin
+  cp -p build/artifacts-$system-$arch/kuma-tcp-echo/kuma-tcp-echo $dest_dir/bin
+  cp -p pkg/config/app/kuma-cp/kuma-cp.defaults.yaml $dest_dir/conf/kuma-cp.conf
+
+  cp tools/releases/templates/* $dest_dir
+
+  tar -czf build/artifacts-$system-$arch/kuma-$system-$arch.tar.gz -C $dest_dir .
```

Comment:
when I run `./tools/releases/distros.sh --package --version 1.2.3` on my machine, I get only 2 `*.tar.gz` files in the end:

* `build/artifacts-darwin-amd64/kuma-darwin-amd64.tar.gz`
* `build/artifacts-linux-amd64/kuma-linux-amd64.tar.gz`

is it an intended behaviour ?

I would expect to have

* `kuma-*-ubuntu.tar.gz`
* `kuma-*-debian.tar.gz`
* `kuma-*-rhel.tar.gz`
* `kuma-*-centos.tar.gz`
* `kuma-*-darwin.tar.gz`

Reply:
This must no longer be the case. It was, before we included Envoy in the tarballs.